lib/vmc-draas-client-bindings/index.html

```diff
@@ -1 +1 @@
-<a href='vmc_draas_client_bindings-1.18.0-py2.py3-none-any.whl'>vmc_draas_client_bindings-1.18.0-py3.py3-none-any.whl</a><br />
+<a href='vmc_draas_client_bindings-1.19.0-py2.py3-none-any.whl'>vmc_draas_client_bindings-1.19.0-py3.py3-none-any.whl</a><br />
```

Comment:
Isn't 1.19 occupying the version that is supposed to be used with VMC 1.19 release? Should we use 1.18.1 instead?
Also once this is merged, are we going to tag/release the hash?

Reply:
These are individual package versions, nothing to do with VMC release versions. We can create a new release tag for VMC DRaaS, but haven't been done traditionally just for DRaaS API changes.